pkg/apis/kf/v1alpha1/space_defaults.go

```diff
@@ -50,7 +53,9 @@ func (k *SpaceSpecBuildpackBuild) SetDefaults(ctx context.Context) {
 
 // SetDefaults implements apis.Defaultable
 func (k *SpaceSpecExecution) SetDefaults(ctx context.Context) {
-	// XXX: currently no defaults to set
+	if len(k.Domains) == 0 {
+		k.Domains = append(k.Domains, DefaultDomain)
```

Comment:
Now that we have two things in here, WDYT about using a configmap out of the kf space to do global configuration of things like the default builder image and domains?

If that's a separate PR that's okay.

Reply:
I like that idea, but I'm thinking it should be a different PR.

Reply:
https://github.com/google/kf/issues/396